AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/editor_python_test_tools/hydra_test_utils.py

```diff
@@ -45,18 +45,23 @@ def launch_and_validate_results(request, test_directory, editor, editor_script,
     :param cfg_args: Additional arguments for CFG, such as LevelName.
     :param timeout: Length of time for test to run. Default is 60.
     :param log_file_name: Name of the log file created by the editor. Defaults to 'Editor.log'
+    :param enable_prefab_system: Specifies whether to use enable prefab system or not. Defaults to False.
     """
     test_case = os.path.join(test_directory, editor_script)
     request.addfinalizer(lambda: teardown_editor(editor))
     logger.debug("Running automated test: {}".format(editor_script))
     editor.args.extend(["--skipWelcomeScreenDialog", "--regset=/Amazon/Settings/EnableSourceControl=false", 
-                        "--regset=/Amazon/Preferences/EnablePrefabSystem=false", run_python, test_case,
+                        run_python, test_case,
                         f"--pythontestcase={request.node.originalname}", "--runpythonargs", " ".join(cfg_args)])
     if auto_test_mode:
         editor.args.extend(["--autotest_mode"])
     if null_renderer:
         editor.args.extend(["-rhi=Null"])
-
+    if enable_prefab_system:
```

Comment:
This will now always override the prefab state instead of using the one set in the project. I think it would be better to just wrap this _run_prefab_test() that sets --regset=/Amazon/Preferences/EnablePrefabSystem=True.